### PR TITLE
Cleaned text and image data for tesseract model

### PR DESCRIPTION
**Pull request by techthumb1**
# Description

Process involved using previously gathered pages of hand-written text to serve as the data. Each file is sectioned by page and then line, which was done by manually taking clips of the hand-written text and label as an image and a corresponding text file. Data will be processed and increase the readability of the image. Providing an extensive amount of training data to the model will provide a wider path in tuning the model and increasing accuracy.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

https://www.loom.com/share/5d3e9c9fbdbb493c880452eff0966fed?sharedAppSource=personal_library

### CODE REVIEW

**Review by Fellylove — APPROVED** [review]
Thanks for doing a great job getting this data ready.

It looks good. There are just a few things that you may want to correct.

- data/storysquad-ground-truth-51/5125/51-5125-1-27.gt.txt: It is "Faintrd" NOT "fainted"

- data/storysquad-ground-truth-51/5126/51-5126-1-20.gt.txt: It is "Victory"  NOT "awesome"
-data/storysquad-ground-truth-51/5130/51-5130-3-18.gt.txt: It reads "Mu wife" NOT "my wife"

- data/storysquad-ground-truth-51/5131/51-5131-1-19-2.gt.txt: Remove the 51-5131-1-19-1.gt.txt from the text

- data/storysquad-ground-truth-51/5131/51-5131-1-20-1.gt.txt: Remove "s" from text 
It is "future. One day" NOT "future. One day s"

- data/storysquad-ground-truth-51/5132/51-5132-1-4.png: Make a note that it was not preprocessed like the others
- data/storysquad-ground-truth-51/5132/51-5132-2-4.gt.txt and data/storysquad-ground-truth-51/5132/51-5132-2-4.png are mismatched
- data/storysquad-ground-truth-51/5132/51-5132-2-5.png and data/storysquad-ground-truth-51/5132/51-5132-2-5.gt.txt are mismatched
- data/storysquad-ground-truth-51/5132/51-5132-2-6.gt.txt and data/storysquad-ground-truth-51/5132/51-5132-2-6.png are mismatched
- data/storysquad-ground-truth-51/5132/51-5132-2-7.gt.txt and
data/storysquad-ground-truth-51/5132/51-5132-2-7.png are mismatched

- data/storysquad-ground-truth-51/5132/51-5132-2-7.gt.txt and
data/storysquad-ground-truth-51/5132/51-5132-2-7.png are mismatched

- The mismatch starts here data/storysquad-ground-truth-51/5132/51-5132-2-4.gt.txt. There is no corresponding png for that text

**Review by cryptobellum — APPROVED** [review]
Your work was very thorough and extensive. Good job on this foundational task!

**Comment by techthumb1** [comment]
Thank you for the approval and the advice. I will make the necessary
changes per your suggestion, thanks again!
